BubbleSorting/ShowingConsole.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BubbleSorting
+{
+    /// <summary>
+    /// Class of display on console
+    /// </summary>
+    class ShowingConsole
+    {
+        /// <summary>
+        /// Display on console
+        /// </summary>
+        /// <param name="jaggedArray">Jagged array</param>
+        public static void ShowConsole(int[][] jaggedArray)
+        {
+            for (int i = 0; i < jaggedArray.Length; i++)
+            {
+                System.Console.Write("Row number {0}: ", i);
```

Comment:
a system namespace was defined in a using section
just Console.Writeline enough